### DISCUSSION TITLE
the directory is world deletable

### DISCUSSION THREAD

**Discussion by govindptl** [antiden/macOS-cursors-for-Windows#55] [Q&A]
![image](https://user-images.githubusercontent.com/83940489/190561225-2247a532-18c3-439c-9fb0-485c14879720.png)
please check the ss.

i haven't tried clicking it...🤷‍♂️

**Comment by antiden** [comment]
Hi,
This is standart github behavior, you can delete folder from any public repo, but first, repo will be forked to your accaunt, and second original repo will remain not touched.

Thx!

**Reply by InternetAstronaut** [reply]
Should've been marked as answered by now... 🤦
